### PR TITLE
feat: [GTM-658]:  update CI, FF, CCM usage cards

### PR DESCRIPTION
**Pull request by nanaxuHarness**
##### Summary:
this is to 
1) update CI, FF, CCM usage cards
2) filter out CD community on plan page
3) display `Services` or `Services Instances` for CD depending on cdLicenseType

##### Jira Links:

https://harness.atlassian.net/browse/PLG-658

##### Screenshots:

before:

should not display `community` for CD plans page:
Screen Shot 2022-02-16 at 11 59 36 AM
`Service Reliability` format is off:
Screen Shot 2022-02-16 at 12 00 11 PM
CI, FF, CCM usage cards need to be updated:
Screen Shot 2022-02-16 at 11 59 21 AM
Screen Shot 2022-02-16 at 11 59 54 AM
Screen Shot 2022-02-16 at 12 00 05 PM

after:
`community` is filtered out:
Screen Shot 2022-02-16 at 1 03 03 PM
`Service Reliability` format is corrected:
Screen Shot 2022-02-16 at 1 02 52 PM
CI, FF, CCM usage cards are updated:
FF:
Screen Shot 2022-02-16 at 1 03 27 PM
Screen Shot 2022-02-16 at 1 03 47 PM
Screen Shot 2022-02-16 at 1 03 41 PM
Screen Shot 2022-02-16 at 1 03 33 PM
CI:
Screen Shot 2022-02-16 at 1 03 14 PM
CCM:
Screen Shot 2022-02-16 at 1 04 02 PM
Screen Shot 2022-02-16 at 1 04 07 PM

You can use the following comments to re-trigger PR Checks
- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by nanaxuHarness** [comment]
retrigger jest

**Comment by nanaxuHarness** [comment]
retrigger jest

**Review by abhinavrastogi-harness — APPROVED** [review]

**Comment by nanaxuHarness** [comment]
trigger cypress

**Review by tan-nhu — APPROVED** [review]

**Comment by mtran7** [comment]
Vertical align Billed with yearly/monthly

**Comment by mtran7** [comment]
Also would spell out "FF" in Developers are the users of Harness FF

**Comment by mtran7** [comment]
And would say "An Active Developer is an active..." To follow the tooltip with the label

File: src/modules/10-common/strings/strings.en.yaml [hunk 1]
```diff
@@ -400,16 +400,21 @@ subscriptions:
...
-    last60days: Last 60 days
+    last30days: Last 30 days
     services: Active Services
     srvcInst: Active Service Instances
     cdUsers: Active CD Users
-    ciUsers: Active CI Users
-    ffUsers: Feature Flags Developers
+    ciUsers: Active CI Developers
+    ciTooltip: A Developer is an active code committer observed in the builds managed by Harness CI, over last 30 days.
+    ffActiveUserTootip: Monthly Active Users (MAUs) are only applicable to client side applications and are defined as unique end users evaluating flags from client-side SDKs (JavaScript, iOS, Android, C/C++, React, etc.) over a 30-day period.
```
Comment (mtran7) [inline src/modules/10-common/strings/strings.en.yaml:409]: Has Docs looked at this? Sounds a bit wordy

File: src/modules/10-common/strings/strings.en.yaml [hunk 2]
```diff
@@ -400,16 +400,21 @@ subscriptions:
...
+    ffDeveloperTooltip: Developers are the users of Harness FF module, that actually engage with the product and use it to create/edit/delete Feature Flags and or Targets/Segments.
```
Comment (mtran7) [inline src/modules/10-common/strings/strings.en.yaml:410]: Would change "actually" to "have". Also would have @michaelcretzman review

**Comment by nanaxuHarness** [comment]
> Vertical align Billed with yearly/monthly

i'll update the changes into this story: https://harness.atlassian.net/browse/PLG-686
